### PR TITLE
Switch between podman or docker

### PR DESCRIPTION
**Pull request by daviddavis**
[noissue]

### CODE REVIEW

**Review by dkliban — APPROVED** [review]

File: generate.sh [hunk 1]
```diff
@@ -3,6 +3,13 @@ if [ $# -eq 0 ]; then
     exit 1
 fi
 
+if command -v docker > /dev/null
```
Comment (mikedep333) [inline generate.sh:6]: Technically it should be the other way around.

podman installations often include a `docker` symlink.

This does not break the current code, but could break future logic that is different for the 2 container runtimes.

**Review by mikedep333 — APPROVED** [review]

File: generate.sh (hunk 1 above)
Comment (daviddavis) [reply generate.sh:6]: I had it the other way around and the CI failed:

https://github.com/pulp/pulp-openapi-generator/runs/2908552918

Github Actions appears to have both podman and docker installed. I'm not sure why the bindings are generating correctly with docker in Github Actions but not podman.

Comment (mikedep333) [reply generate.sh:6]: Interesting, sounds good.

Comment (mdellweg) [reply generate.sh:6]: Also if podman brings the symlink, the result of this check does not matter at all...

**Pull request merged**